src/main/java/org/killbill/billing/plugin/adyen/client/AdyenConfigProperties.java

```diff
@@ -84,6 +88,14 @@ public AdyenConfigProperties(final Properties properties) {
         this.hmacSecrets = properties.getProperty(PROPERTY_PREFIX + "hmac.secret");
         refillMap(secretMap, hmacSecrets);
 
+        final String hmacAlgorithm = properties.getProperty(PROPERTY_PREFIX + "hmac.algorithm");
+        if (hmacAlgorithm != null) {
```

Comment:
Nit: `this.hmacAlgorithms = Objects.firstNonNull(properties.getProperty(PROPERTY_PREFIX + "hmac.algorithm"),  DEFAULT_HMAC_ALGORITHM);`